### PR TITLE
Repo names changed again

### PR DESCRIPTION
**Pull request by dpoirier**

### CODE REVIEW

File: conf/pillar/testing/env.sls [hunk 1]
```diff
@@ -3,8 +3,8 @@ environment: testing
 domain: serviceinfo-testing.caktusgroup.com
 
 repo:
-  url: https://github.com/theirc/ServiceInfo-project.git
-  branch: open_source_project
+  url: https://github.com/theirc/ServiceInfo.git
+  branch: develop
```
Comment (vkurup) [inline conf/pillar/testing/env.sls]: Is the `testing` environment still supported? I remember seeing the `settings/testing.py` file get deleted in the project repo.

File: conf/salt/margarita.sls [hunk 2]
```diff
@@ -5,7 +5,7 @@ git-install:
 project_repo:
   git.latest:
     - name: https://github.com/caktus/margarita.git
-    - rev: 1.0.1
+    - rev: 1.0.0
```
Comment (vkurup) [inline conf/salt/margarita.sls]: This seems like a bump downwards. Was this supposed to be 1.0.2?

**Comment by dpoirier** [comment]
Updated - just fix the repo names and don't change anything else.

**Comment by vkurup** [comment]
:+1:

**Pull request merged**